### DISCUSSION TITLE
Mac System Settings for RustDesk (Remote Computer)

### DISCUSSION THREAD

**Discussion by JonHaverstick** [rustdesk/rustdesk#6987] [Q&A]
Hi Folks - 

We just set up RustDesk on my Mom's 2023 MacBook Air (Sonoma 14.2.1) so that I can assist her remotely with technical issues.  We had it running on her old computer, but when we tried to install and connect today, I could see her screen, but not interact / control / click on anything remotely.  We enabled screen recording in her privacy settings, and even rebooted her computer.  Still no "click ability" on the remote computer from my end.  Is there any other system setting we need to enable / change on the remote computer in order for me to actually interact with her Mac? 

Thanks!

**Comment by tmikkonen** [comment]
I realise the question is from 10 months back, but just in case anyone is still wondering:
_Privacy & Security -> Accessibility_ should be enabled for RustDesk, too.
